app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -1039,6 +1056,15 @@ class MainViewModel @Inject constructor(
         snackbarManager.postValue(appContext.getString(snackbarTitleRes))
     }
 
+    private fun isMessageTheLastInFolder(threadUid: String): Boolean {
```

Comment:
The name is confusing.

Reply:
is this name better : threadHasOnlyOneMessage